app/views/home/index.html.haml

```diff
@@ -163,7 +183,7 @@
       .home-support-items
         - $HOME_PAGE_LOGOS[:with_the_collaboration_of].each do |logo|
           %a{href:logo[:url], target: "_blanc"}
-            %img{src: asset_path(logo[:img_src])}
+            %img{src: asset_path(logo[:img_src]), class: (logo[:img_src] == 'logos/collaboration/ontoportal.png' ? 'custom-logo-class' : '')}
```

Comment:
This solution is not generic enough 

set the same width and height for all the images (and object-fit: scale-down)

Reply:
@syphax-bouazzouni  I tried this solution, but the display of the logos wasn't clear, and it was pixelated (for example the logo of inrae)

Reply:
did you use add the `object-fit: scale-down` style, so that is not pixilated.
You can see also the other options https://developer.mozilla.org/fr/docs/Web/CSS/object-fit

Reply:
done, thank you
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/d532bd43-f5bc-4e32-989b-8c11890da0b5)

Reply:
Great